### DISCUSSION TITLE
tutorials/workshop-summit24-prereqs/02-w06-router

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#169] [course feedback]
# tutorials/workshop-summit24-prereqs/02-w06-router

https://www.apollographql.com/tutorials/workshop-summit24-prereqs/02-w06-router

**Comment by Plygon** [comment]
I get this error every time I try to run `docker compose up -d`:

`Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "/etc/loki/loki.sh": stat /etc/loki/loki.sh: no such file or directory: unknown`.

I've tried restarting my computer, running in Windows, running in WSL, deleting all images, and reinstalling docker

**Reply by Geal** [reply]
Is it only happening for the loki container?

**Reply by Plygon** [reply]
That's the only error that I'm seeing  in the console, but localhost:8080 is not starting up.

But I got it working on another laptop, so I'm good now

**Reply by Geal** [reply]
Alright, good to know! If loki is not working,  that would not be a blocker for the workshop anyway
